index.md

```diff
@@ -70,6 +70,15 @@ like creating / deleting databases and loading schemas from files.
 
 * [GitHub repository](https://github.com/cloudspannerecosystem/wrench)
 
+### yo
+
+yo is a command-line tool to generate Go code for Cloud Spanner.
+yo uses database schema to generate code by using Information Schema.
+yo runs SQL queries against tables in INFORMATION_SCHEMA to fetch metadata for a database,
+and applies the metadata to Go templates to generate code/models to acccess Cloud Spanner.
```

Comment:
Thank you very much!

Suggesting a slight rewording, but please feel free to disregard in case the following isn't accurate:

yo is a command-line tool to generate database-specific code to access Cloud Spanner through Go objects instead of SQL queries. It does so by fetching schema metadata for a database by querying the [information schema](https://cloud.google.com/spanner/docs/information-schema) and applies Go templates to generate code / models.

Reply:
@lgruen Thanks, just use your suggestion!

Reply:
Awesome, thanks!